packages/aws-cdk-lib/aws-apigatewayv2/README.md

```diff
@@ -2,20 +2,24 @@
 
 ## Table of Contents
 
-- [Introduction](#introduction)
-- [HTTP API](#http-api)
-  - [Defining HTTP APIs](#defining-http-apis)
-  - [Cross Origin Resource Sharing (CORS)](#cross-origin-resource-sharing-cors)
-  - [Publishing HTTP APIs](#publishing-http-apis)
-  - [Custom Domain](#custom-domain)
-  - [Mutual TLS](#mutual-tls-mtls)
-  - [Managing access to HTTP APIs](#managing-access-to-http-apis)
-  - [Metrics](#metrics)
-  - [VPC Link](#vpc-link)
-  - [Private Integration](#private-integration)
-- [WebSocket API](#websocket-api)
-  - [Manage Connections Permission](#manage-connections-permission)
-  - [Managing access to WebSocket APIs](#managing-access-to-websocket-apis)
+- [AWS APIGatewayv2 Construct Library](#aws-apigatewayv2-construct-library)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [HTTP API](#http-api)
+    - [Defining HTTP APIs](#defining-http-apis)
+    - [Cross Origin Resource Sharing (CORS)](#cross-origin-resource-sharing-cors)
+    - [Publishing HTTP APIs](#publishing-http-apis)
+    - [Custom Domain](#custom-domain)
+    - [Mutual TLS (mTLS)](#mutual-tls-mtls)
+    - [Managing access to HTTP APIs](#managing-access-to-http-apis)
+    - [Metrics](#metrics)
+    - [VPC Link](#vpc-link)
+    - [Private Integration](#private-integration)
+    - [Generating ARN for Execute API](#generating-arn-for-execute-api)
```

Comment:
The formatting here is off. The only diff in this part of the readme should be adding the `- [Generating ARN for Execute API](#generating-arn-for-execute-api)` point after Private Integration, as a bullet under the `- [HTTP API](#http-api)` bullet.

See the difference between the [main README](https://github.com/aws/aws-cdk/tree/main/packages/aws-cdk-lib/aws-apigatewayv2) and [yours](https://github.com/badmintoncryer/aws-cdk/tree/23301-addArnForExecuteAPI/packages/aws-cdk-lib/aws-apigatewayv2#table-of-contents)